src/rockstor/storageadmin/models/disk.py

```diff
@@ -18,14 +18,18 @@
 
 from django.db import models
 from storageadmin.models import Pool
-from system.osi import get_disk_power_status, get_dev_byid_name, \
-    read_hdparm_setting, get_disk_APM_level
+from system.osi import get_disk_power_status, read_hdparm_setting, \
+    get_disk_APM_level
 
 
 class Disk(models.Model):
     """Pool can be null for disks that are not part of any pool currently"""
     pool = models.ForeignKey(Pool, null=True, on_delete=models.SET_NULL)
-    """typically sda, sdb etc.. max_length = 0 supports 100s of disks"""
+    """ Previously the name field contained sda, sdb etc..
+    Revised to contain device names for use with the udev created links at
+    /dev/disk/by-id/ which in turn are symlinks to sda, sdb etc.
+    eg ata-QEMU_HARDDISK_QM00005 ie mostly derived from model and serial number.
+    """
```

Comment:
I support this change, but strongly prefer that primary user facing name of a Disk should be the more readable/humanized one such as sda, sdb etc.. Since they are not persistent, it makes sense to store the persistent by-id name in the db instead. However, I'd like to suggest a property that retrieves the readable name during runtime as the model is feteched.

This also requires some UI changes, wherein the readable name is presented to the admin for various management matters like creating a Pool, adding disks etc.., with the Disk.name in an additional column or in a tooltip. This way, we achieve a pleasant user experience while not compromising the true identity of the Disk.